### PR TITLE
Add section with citations in the main website

### PR DESCRIPTION
**Pull request by beatrizserrano**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,17 +0,0 @@
-vendor
```
Comment (bgruening) [inline .gitignore:1]: please do not remove this file ...

File: about.md [hunk 2]
```diff
@@ -40,7 +40,7 @@ To dig deeper, **interactive visualizations** are available:
 
 # <i class="fa fa-cogs"></i> Tools for any Bioinformatics Analysis
 
-More than 1,000 bioinformatics tools can be accessed via [https://usegalaxy.eu](https://usegalaxy.eu), covering most of the bioinformatics topics:
+More than 2500 bioinformatics tools can be accessed via [https://usegalaxy.eu](https://usegalaxy.eu), covering most of the bioinformatics topics:
```
Comment (bgruening) [inline about.md:43]: maybe scientific tools?

Comment (beatrizserrano) [reply about.md:43]: yeah!

File: citations.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+---
+layout: default
+---
+
+# Galaxy EU citations
```
Comment (bgruening) [inline citations.md:5]: Maybe here a little bit more text? That this is a list of papers that cite the European Galaxy instance ....

File: environment.yml [hunk 4]
```diff
@@ -4,79 +4,75 @@ channels:
...
-  - rb-tzinfo-data=1.2019.3=hc42de5b_0
-  - rb-unicode-display_width=1.6.0=hc42de5b_0
-  - rb-yell=2.2.2=hc42de5b_0
-  - readline=8.0=hf8c457e_0
-  - ruby=2.6.5=h305c5c1_0
-  - tk=8.6.10=hed695b0_0
-  - xz=5.2.4=h14c3975_1001
-  - yaml=0.1.7=h14c3975_1001
-  - zlib=1.2.11=h516909a_1006
+  - _libgcc_mutex=0.1
```
Comment (bgruening) [inline environment.yml:7]: that should be removed I think

**Pull request merged**